extras/indexing/src/main/java/org/apache/rya/indexing/accumulo/ConfigUtils.java

```diff
@@ -66,21 +67,21 @@
 import com.google.common.collect.Lists;
 
 /**
- * A set of configuration utils to read a Hadoop {@link Configuration} object and create Cloudbase/Accumulo objects.
- * Soon will deprecate this class.  Use installer for the set methods, use {@link RyaDetails} for the get methods.
- * New code must separate parameters that are set at Rya install time from that which is specific to the client.
- * Also Accumulo index tables are pushed down to the implementation and not configured in conf.
+ * A set of configuration utils to read a Hadoop {@link Configuration} object
+ * and create Cloudbase/Accumulo objects. Soon will deprecate this class. Use
+ * installer for the set methods, use {@link RyaDetails} for the get methods.
+ * New code must separate parameters that are set at Rya install time from that
+ * which is specific to the client. Also Accumulo index tables are pushed down
+ * to the implementation and not configured in conf.
  */
 public class ConfigUtils {
     private static final Logger logger = Logger.getLogger(ConfigUtils.class);
 
-    public static final String CLOUDBASE_TBL_PREFIX = "sc.cloudbase.tableprefix";
-    public static final String CLOUDBASE_AUTHS = "sc.cloudbase.authorizations";
-    public static final String CLOUDBASE_INSTANCE = "sc.cloudbase.instancename";
-    public static final String CLOUDBASE_ZOOKEEPERS = "sc.cloudbase.zookeepers";
-    public static final String CLOUDBASE_USER = "sc.cloudbase.username";
-    public static final String CLOUDBASE_PASSWORD = "sc.cloudbase.password";
-
+    public static final String CLOUDBASE_INSTANCE = AccumuloRdfConfiguration.CLOUDBASE_INSTANCE;
```

Comment:
You should mark all of the moved static variables as deprecated and indicate where they have been moved to in the deprecation documentation.